library/index.md

```diff
@@ -43,12 +43,12 @@ permalink: /library/index.html
     </section>
     <section class="container full">
         <div class="info-block text-adapt">
-            <h2>{% t library.magazines %}</h2>
+            <h2>{% t library.newsletters %}</h2>
             <div>
-                <h3><a href="{{ site.baseurl_root }}/library/Revuo-2017-Q4.pdf">Revuo Monero Q4 2017</a></h3>
-                    <p>{% t library.revuoq4p %}</p>
-                <h3><a href="{{ site.baseurl_root }}/library/Monero-Revuo-3Q-2017.pdf">Revuo Monero Q3 2017</a></h3>
-                    <p>{% t library.revuoq3p %}</p>
+                <h3><a href="https://localmonero.co/the-monero-standard">The Monero Standard Series</a></h3>
+                  <p>{% t library.themonerostandard %}</p>
+                <h3><a href="https://monero.observer/tag/blitz/">Monero Observer Blitz Series</a></h3>
```

Comment:
Same as above: should be `Monero Observer Blitz`, or `series` should be translatable.